src/styles/main.scss

```diff
@@ -1 +1,72 @@
-// put code here
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  align-items: center;
+  justify-content: center;
+}
+
+.clock {
+  margin: 100px 400px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 600px;
+  height: 600px;
+  border-radius: 100%;
+  border-color: black;
+  border: dotted 4px;
+
+  &__centr {
+    position: relative;
+    width: 60px;
+    height: 60px;
+    border: 1px;
+    border-radius: 100%;
+    background-color: yellow;
+  }
+
+  &__hours {
+    margin-left: 190px;
+    position: absolute;
+    background-color: blue;
+    width: 210px;
+    height: 30px;
+
+    transform-origin: left center;
```

Comment:
animation-duration should be removed as it's already defined in the animation shorthand property.